src/Controller/Admin/TournamentController.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace App\Controller\Admin;
+
+use App\Entity\Participant;
+use App\Entity\Player;
+use App\Entity\Tournament;
+use App\Form\Type\AddParticipantType;
+use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+use Symfony\Component\HttpFoundation\Request;
+
+class TournamentController extends Controller
+{
+    public function showPlayers(Request $request, $title)
```

Comment:
rename to showPlayersAction
It will be clear if the method is action or not, and it's little bit weird